resources/netregexes.ts

```diff
@@ -129,6 +131,7 @@ const parseHelper = <T extends LogDefinitionTypes>(
 };
 
 export default class NetRegexes {
+  public static lang: Lang = 'en';
```

Comment:
I agree with this approach too.  I think it's kind of sad to have a static variable here controlling things, but it seems like it would be obnoxious to change all NetRegexes call to explicitly take in a language variable.  And how would that even work for things like trigger files?  I think this is probably the best approach given all the tradeoffs?

Reply:
The regexes/netregexes.ts files could instead export an instance of Regexes/NetRegexes but at that point it's basically the same thing (singleton vs static class).